assets/src/edit-story/app/media/utils/useUploadVideoFrame.js

```diff
@@ -54,8 +63,12 @@ function useUploadVideoFrame() {
         featured_media: posterId,
         post: storyId,
       });
-      const newState = { posterId, poster };
-      setProperties(elementId, newState);
+      if (elementId) {
+        const newState = { resource: { posterId, poster } };
```

Comment:
Wouldn't that completely override the existing resource, including the existing fields?

Reply:
Fixed in https://github.com/google/web-stories-wp/pull/553/commits/7b5d496e7b8cb77471e278243c4633d8b462d8e8